src/io/output_mesh.hh

```diff
@@ -84,14 +90,15 @@ public:
     /// Gives iterator to the LAST element of the output mesh.
     OutputElementIterator end();
     
-    /// Creates the output mesh identical to the orig mesh.
-    virtual void create_mesh()=0;
-
-    /// Creates refined mesh.
-    virtual void create_refined_mesh()=0;
+    /**
+     * Creates sub mesh containing only local part of original (computation) mesh.
+     */
+    void create_sub_mesh();
```

Comment:
add: TODO: should be replaced by local part of future parallel computational mesh.